pkg/cloudprovider/snapshot_service.go

```diff
@@ -67,8 +67,8 @@ func NewSnapshotService(blockStorage BlockStorageAdapter) SnapshotService {
 	}
 }
 
-func (sr *snapshotService) CreateVolumeFromSnapshot(snapshotID string, volumeType string, iops *int64) (string, error) {
-	volumeID, err := sr.blockStorage.CreateVolumeFromSnapshot(snapshotID, volumeType, iops)
+func (sr *snapshotService) CreateVolumeFromSnapshot(snapshotID string, volumeType string, volAZ string, iops *int64) (string, error) {
```

Comment:
It's a nit... but you can have `snapshotID string, volumeType, volAZ string, iops *int64` here

Reply:
Agree. Didn't want to change the style of the original code. But, will change.